pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -2737,6 +2729,7 @@ public void cancelPublishBufferLimiting() {
             if (e.getCause() instanceof BrokerServiceException) {
                 error = BrokerServiceException.getClientErrorCode(e.getCause());
             }
+            log.error("some thing error.", e);
```

Comment:
Please use more meaningful error log.

Reply:
That is a common method for "get error code from CompetableFuture", could not append more clear description

Reply:
Why do we need this log here?

Reply:
already deleted